problems/4.arrays.js

```diff
@@ -12,16 +12,18 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
-}
+// function duplicateArray(arr) {
+//   let arr = arr1;
+//   return arr.concat(arr1)
+// }
 
 /** Removes the first 3 elements from an array. If the array has fewer than 3 elements, return an empty array.
 * @param {any[]} arr - The input array
 * @returns {any[]} - The array with the first 3 elements removed
 */
-function removeFirstThreeElements() {
-
+function removeFirstThreeElements(arr) {
+  let arr = [];
+  return arr.shift(0), arr.shift(1), arr.shift(2)
```

Comment:
revisit shift() on MDN. Does not take an argument. Returns the element removed. YOu want to return the arr after it's been shifted. Don't redeclare on line 25.